java/flight/flight-integration-tests/pom.xml

```diff
@@ -45,6 +45,10 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+	<dependency>
```

Comment:
nit: this is mis-indented

Reply:
There's also tabs, IntelliJ got into the catnip again...